sql/core/src/main/scala/org/apache/spark/sql/execution/ExternalAppendOnlyUnsafeRowArray.scala

```diff
@@ -204,20 +199,37 @@ private[sql] class ExternalAppendOnlyUnsafeRowArray(
     }
   }
 
-  private[this] class SpillableArrayIterator(
+  private[this] class MergerIterator(
```

Comment:
nit: How about `MergerIterator` -> `SpilledArrayMergeIterator`?

Reply:
Done. Thank you